cmd/otel-agent/subcommands/run/command.go

```diff
@@ -77,6 +78,7 @@ func runOTelAgentCommand(_ context.Context, params *subcommands.GlobalParams, op
 	err := fxutil.Run(
 		forwarder.Bundle(),
 		corelogimpl.Module(),
+		otelflarefx.Module(),
```

Comment:
Should we run flare provider in core Agent? Whats the need to run in the OTel Agent

Reply:
That was a mistake, it should be in the core agent instead. Moved it in 80cb96d0c036e1eb67401128598950a72b756518